ui/pages/confirmations/components/edit-gas-fee-popover/network-statistics/status-slider/status-slider.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable @metamask/design-tokens/color-no-hex*/
```

Comment:
Remove this

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/24089/commits/f7543dd9bbb2a67a9e84a8ed6343ced95b4958bd